deploy/helm/templates/secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.envSecret.create -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.envSecret.name | default (include "starboard-operator.fullname" .) | quote }}
+  labels:
+    {{- include "starboard-operator.labels" . | nindent 4 }}
+stringData:
+  {{- range $key, $value := .Values.envSecret.stringData }}
+  {{ $key | quote }}: {{ $value | quote }}
+  {{- end }}
```

Comment:
I opted to use `stringData` as the k8s Secret resource's content here, it is the same as `data` but doesn't require us to base64 encode it first. This doesn't make a difference really, except that it is easier for administrators to read/edit the secret resource if they feel like it.